lib/appsignal/config.rb

```diff
@@ -102,6 +102,7 @@ def dsl_config_file?
       :enable_gvl_global_timer => true,
       :enable_gvl_waiting_threads => true,
       :enable_rails_error_reporter => true,
+      :enable_active_support_event_reporter => true,
```

Comment:
Nit: maybe we should make the option specific for logging in case we want to add another reporter that does add events to the transaction events later?

```suggestion
      :enable_active_support_event_log_reporter => true,
```